rules/game_process.tex

```diff
@@ -340,6 +365,21 @@ \subsection{Free Kick}
 Note that in the case of the Pushing free kick the sub-state is activated automatically through the ``Foul''.
 The team who is awarded the free kick (termed the attacking team) has \qty{\FreeKickTime}{\second} to complete the kick.
 
+In Champions Cup, the referee communicates which team is to take the free kick via a visual gesture.
+The GameController packet will tell the robots which free kick sub-state is active, but not who is to kick the ball.
+\unsure{This gesture is entirely tentative and just my (Francesco's) proposal, pending discussion with the TC. The intent is to mirror the FIFA gestures in a simplified way.}
+The gesture consists of raising one arm horizontally and sideways, parallel to the field line, while the other arm remains at rest.
+The raised arm is the one pointing towards the defending team's goal, i.e. the goal of the team that is \textit{not} taking the free kick.
```

Comment:
It would be easier to say that the raised arm is pointing in the direction the free kick has been given (i.e. towards the goal being attacked).

Reply:
My rationale here is that I feel like referring directly to which team is or is not taking the free kick is the most technical and unambiguous way to define the rule in this case, and the more intuitive "defending team" definition is still given first, before the more explicit one.
Furthermore, especially with the indirect kick rule, the robot may perform the free kick in any direction, so saying that the arm points "in the direction the free kick has been given" might be a little odd.